packages/app-contracts/src/Codes/Code.tsx

```diff
@@ -0,0 +1,171 @@
+// Copyright 2017-2019 @polkadot/app-staking authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { I18nProps } from '@polkadot/ui-app/types';
+import { CodeStored } from '../types';
+
+import React from 'react';
+import styled from 'styled-components';
+import { RouteComponentProps } from 'react-router';
+import { withRouter } from 'react-router-dom';
+import { Button, Card, CodeRow, Forget, Icon } from '@polkadot/ui-app';
+import { withMulti } from '@polkadot/ui-api';
+
+import ABI from '../ABI';
+import RemoveABI from '../RemoveABI';
+
+import contracts from '../store';
+import translate from '../translate';
+
+type Props = I18nProps & RouteComponentProps & {
+  basePath: string,
+  code: CodeStored,
+  showDeploy: (codeHash?: string) => () => void
+};
+
+type State = {
+  isForgetOpen: boolean,
+  isRemoveABIOpen: boolean
+};
+
+const CodeCard = styled(Card)`
+  && {
```

Comment:
... idiot time ... somebody needs to explain `&& { ... }` to me...

Reply:
Adds a dupllicate declaration (.class.class) so just a hacky way of prioritizing it over any other annoying styles - don't remember what this one was for specifically